ci/axis/dask.yaml

```diff
@@ -17,3 +18,6 @@ RAPIDS_VER:
   - '22.02'
 
 excludes:
+  # dask-image gpuCI isn't dependent on RAPIDS
+  - RAPID_VER: '21.12'
+    BUILD_NAME: dask_image
```

Comment:
Just noting that this is here because dask-image doesn't depend on RAPIDS version since we only require CuPy